server/tests/batch_comparison/test_multi_contest_batch_comparison.py

```diff
@@ -0,0 +1,1003 @@
+from typing import Dict, List
```

Comment:
GitHub is auto-collapsing this test file, so leaving a comment so it doesn't get missed